components/Footer/StyledNav.js

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+
+export const StyledNav = styled.div`
```

Comment:
is there a reason why you put the StyledNav in a separate file instead of write the styled component in Nav.js?

Reply:
i think its more structured, thats why :)